iis/datadog_checks/iis/iis.py

```diff
@@ -55,19 +55,26 @@ def get_iishost(self, instance):
         return "iis_host:{}".format(self.normalize(iis_host))
 
     def check(self, instance):
+        # Duplicate refresh_counters from PDHBaseCheck
+        instance_hash = hash_mutable(instance)
+        refresh_counters = is_affirmative(instance.get('refresh_counters', True))
```

Comment:
does it make sense to extract this in the parent class to a method and then have both parent and iis check methods call refresh_counters?

Reply:
It makes it a bit harder to deliver to the customer. In the long run, you're right, though iis is the only case.